guilford_sent_component.yaml

```diff
@@ -200,6 +195,11 @@ entityDefinitions:
       ol.id:
         type: "ol.id"
         column: "CREDITS FOR JAIL DAYS SERVED"
+      ol.type:
+        type: "ol.type"
+        transforms:
+        - !<transforms.ValueTransform>
```

Comment:
looks good! The only thing is this value should say "Credits for jail days served". The other jailtimemodifications below is for the ICC credits.

Reply:
whoops, thanks. fixing.